app-games/src/main/java/com/aptoide/android/aptoidegames/settings/SettingsScreen.kt

```diff
@@ -76,6 +77,8 @@ fun NavGraphBuilder.settingsScreen(
     downloadOnlyOverWifi = downloadOnlyOverWifi,
     isDarkTheme = isDarkTheme,
     acceptedPPAndTC = acceptedPPAndTC,
+    verName = BuildConfig.VERSION_NAME,
```

Comment:
It is the same value as current default value in the `SettingsViewContent` composable 🤔

Reply:
The `SettingsViewContent` composable has the default values "1.2.3" and "123" which should only be used for previews, hence this change. However, app-games should probably have its own versioning system using Gradle properties